releases/release-4.0.13.md

```diff
@@ -0,0 +1,136 @@
+    - 修复 `IndexJoin` 在前缀列索引上计算结果出错的问题 [#23691](https://github.com/pingcap/tidb/pull/23691)
+    - 修复由于 `BINARY` 类型列上排序规则处理不当导致查询结果出错的问题 [#23598](https://github.com/pingcap/tidb/pull/23598)
+    - 修复当 `UPDATE` 语句中存在含 `HAVING` 子句的连接查询时，执行出错的问题 [#23575](https://github.com/pingcap/tidb/pull/23575)
+    - 修复比较类型表达式中使用 `NULL` 常量导致 TiFlash 计算结果出错的问题 [#23474](https://github.com/pingcap/tidb/pull/23474)
+    - 修复 `YEAR` 类型列与字符类型常量比较结果出错的问题 [#23335](https://github.com/pingcap/tidb/pull/23335)
+    ?- 修复 `session.group_concat_max_len` 被设置为 `group_concat` 时执行出错的问题 [#23257](https://github.com/pingcap/tidb/pull/23257)
+    - 修复 `TIME` 类型列上使用 `BETWEEN` 表达式计算结果出错的问题 [#23233](https://github.com/pingcap/tidb/pull/23233)
+    - 修复 `DELETE` 语句中出现的权限检查问题 [#23215](https://github.com/pingcap/tidb/pull/23215)
+    ?- Fix the issue that query return wrong query result for decimal type [#23196](https://github.com/pingcap/tidb/pull/23196)
+    - 修复往 `DECIMAL` 类型列插入数据时解析出错的问题 [#23152](https://github.com/pingcap/tidb/pull/23152)
+    - 修复 `USE_INDEX_MERGE` hint 无法生效的问题 [#22924](https://github.com/pingcap/tidb/pull/22924)
+    - 修复使用 `ENUM` 或 `SET` 类型列作为 `WHERE` 过滤条件时，查询结果出错的问题 [#22814](https://github.com/pingcap/tidb/pull/22814)
+    - 修复 Clustered Index 与 New Collation 同时使用时，查询结果出错的问题 [#21408](https://github.com/pingcap/tidb/pull/21408)
+    - 修复 `enable_new_collation` 开启时，`ANALYZE` 出错的问题 [#21299](https://github.com/pingcap/tidb/pull/21299)
+    - 修复视图处理默认 ROLE 时，未正确处理相关 DEFINER 的问题 [#24531](https://github.com/pingcap/tidb/pull/24531)
+    - 修复取消 DDL Job 时卡住的问题 [#24445](https://github.com/pingcap/tidb/pull/24445)
+    ?- 修复 `concat` 函数处理排序规则错误的问题 [#24300](https://github.com/pingcap/tidb/pull/24300)
+    - 修复当 `SELECT` 域中包含 `IN` 子查询且子查询外侧表含有空值元组时，查询结果出错的问题 [#24022](https://github.com/pingcap/tidb/pull/24022)
+    - 修复逆序扫表时，TiFlash 被优化器错误选用的问题 [#23974](https://github.com/pingcap/tidb/pull/23974)
+    - 修复点查的返回结果中，列名与 MySQL 不一致的问题 [#23970](https://github.com/pingcap/tidb/pull/23970)
+    ?- 修复 `show table status` for the database with upper-cased name. [#23958](https://github.com/pingcap/tidb/pull/23958)
+    - 修复不同时拥有 `INSERT` 及 `DELETE` 权限的用户可以执行 `REPLACE` 操作的问题 [#23938](https://github.com/pingcap/tidb/pull/23938)
+    - 修复由于未正确处理排序规则导致的 `concat`/`make_set`/`insert` 表达式计算结果出错的问题 [#23878](https://github.com/pingcap/tidb/pull/23878)
+    - 修复在含有 RANGE 分区的表上查询时，查询崩溃的问题 [#23689](https://github.com/pingcap/tidb/pull/23689)
+    - 修复如下问题：在旧版本的集群中，若 `tidb_enable_table_partition` 被设置为 `false`，含有分区的表会被当作普通表处理。此时由旧版本升级至新版本时，在该表上执行 `batch point get` 查询会导致连接崩溃 [#23682](https://github.com/pingcap/tidb/pull/23682)
+    - 修复 TiDB 被配置监听 TCP 连接及 UNIX 套接字时，TCP 连接中远程主机未被正确验证的问题 [#23513](https://github.com/pingcap/tidb/pull/23513)
+    - 修复由于非默认的排序规则导致查询结果出错的问题 [#22923](https://github.com/pingcap/tidb/pull/22923)
+    ?- 修复 Grafana 面板中的 Coprocessor Cache 不显示数据的问题 [#22617](https://github.com/pingcap/tidb/pull/22617)
```

Comment:
@XuHuaiyu PTAL. Is "Coprocessor Cache" a panel of Grafana?

Reply:
Yes